MASH-FRET/source/simulations/arb2phtn.m

```diff
@@ -1,25 +1,11 @@
-function dat = arb2phtn(varargin)
+function dat = arb2phtn(dat,offset,K,eta)
```

Comment:
Careful here. This function has tons of dependencies: e.g. updateFields. MASH is not running yet.

Reply:
Should be running by now